data/brands/amenity/restaurant.json

```diff
@@ -496,6 +496,21 @@
         "name": "Bonchon Chicken"
       }
     },
+    {
+      "displayName": "Bone Daddies",
+      "locationSet": {
+        "include": ["gb-lon.geojson"]
+      },
+      "tags": {
+        "amenity": "restaurant",
+        "brand": "Bone Daddies",
+        "brand:wikidata": "Q111204601",
+        "cuisine": "japanese;ramen",
+        "name": "Bone Daddies",
+        "reservation": "no",
```

Comment:
This is `reservation` first time appear in NSI, should we record this?

My opinion is that if every branch of this restaurant refuse reservation, and this is a unique feature, we can add it.

Reply:
On their Twitter account they make a point of saying no bookings https://www.twitter.com/BoneDaddiesLDN